### ISSUE TITLE
Integrate Profile verification for Project owners 

### ISSUE THREAD

**Issue by markoprljic** [Giveth/giveth-2#286]
## ⏰ Urgency/Deadline/Blockers?
We believe this is a major MVP blocker.

## ❓ Why is it important?
Donors who come to the Giveth 

## 🎉 Subtasks

- [ ] 
- [ ] 
- [ ] 

## 🤼‍ Reviewer

@markoprljic @mateodaza @geleeroyale @willyogo

**Comment by mateodaza** [comment]
We are considering [3box](https://3box.io/) for it

**Comment by markoprljic** [comment]
Has this been discussed already?

**Comment by mateodaza** [comment]
small discussion almost a couple of weeks ago, we can bring it up again this sunday or after mvp release

**Comment by markoprljic** [comment]
I was thinking of the following:

1. User signs up, connect with Torus, **cannot Create Project yet**
2. If user click on Create a project we ask them to complete their profile first (popup) 
<img width="386" alt="Image 2021-01-14 at 2 44 11 pm" src="https://user-images.githubusercontent.com/5704285/104598840-53aee580-5677-11eb-80df-a568df897acc.png">
3. Show notification in My Account to complete the profile 
<img width="1156" alt="Image 2021-01-14 at 4 05 16 pm" src="https://user-images.githubusercontent.com/5704285/104608620-624eca00-5682-11eb-8e27-b54f127c6715.png">
4. Open Edit Profile modal (will be standard for later if user wants to change profile info) 
5. Save profile

This is for starters, there are some open questions:
- Do we want to properly verify users before allowing them to Create a project? (their email is already verified by connecting with Torus)
- This is more about social proof and how we want to communicate the credibility of the project owner to the donors. What information is enough to do this?

Let's discuss.

**Comment by markoprljic** [comment]
Proof of humanity, social profile, URL. Show your donors who you are.

**Comment by markoprljic** [comment]
Create a Public Profile page, reuse My Account UI just remove sidebar and other elements on that page.

**Comment by mateodaza** [comment]
Backend mutation needed to make this work. Issue related #431

**Comment by laurenluz** [comment]
There's no option (at least none that I see) for adding/removing/changing profile picture... Can we add that? @mateodaza 

![image](https://user-images.githubusercontent.com/75490971/107159459-bf722e80-6955-11eb-9949-4c276ccab577.png)

Use case: would be useful, for example, as a personal logging in with facebook to perhaps be able to change their profile pic to something more professional

**Comment by jamespfarrell** [comment]
I think this will have to be fast follow @mateodaza ?

**Comment by mateodaza** [comment]
Yep, more details like profile picture should be happening soon as follow up

**Comment by jamespfarrell** [comment]
@laurenluz @markoprljic 
Is the only outstanding requirement on this "adding/removing/changing profile" ?

**Comment by markoprljic** [comment]
Profile picture, yes.

**Comment by markoprljic** [comment]
@jamespfarrell @mateodaza I added two dependencies to this issue.

**Comment by willyogo** [comment]
I love what we have here so far, but without 3box or something equivalent, we don't _really_ have any identity verification/validation. It would be awesome to give projects the ability to verify their twitter, github, or email address via 3box and display these badges on their profile
https://github.com/3box/3box-verifications
![image](https://user-images.githubusercontent.com/26232238/108158176-a0844280-70a1-11eb-8a7d-feffc1830e88.png)

**Comment by mateodaza** [comment]
I wanna consider 3box for everything related to the verification, I'm still not sure how complete it is but I think it can be enough. I'll start investigating

**Comment by mateodaza** [comment]
https://idx.xyz/ this is the 'new' 3box, It's looking pretty nice!
The less info we save from our users the better, if we can solve the profile picture and eventually the verification through this would be perfect.

**Comment by markoprljic** [comment]
Suggestion from @WhyldWanderer to include Trust Level similar to Gitcoin. 

![Image 2021-05-03 at 4 49 49 pm](https://user-images.githubusercontent.com/5704285/116891995-a2d15400-ac2f-11eb-9e39-b91a1ac3a654.jpg)

![Image 2021-05-03 at 4 50 06 pm](https://user-images.githubusercontent.com/5704285/116892016-a8c73500-ac2f-11eb-83e4-19aacf3b0321.jpg)

**Comment by WhyldWanderer** [comment]
Thanks @markoprljic..

I was initially envisioning this more for visualizing project score during creation.
There are different things that you can add to increase your project score and maybe we can use a similar way of showing the weights of different project aspects.
You can boost your project score by percentages depending on what pices you have completed.
Maybe the description is worth 25% if you have links, +5% if you have images or videos +10% etc....

However this may be useful for verifying users identity as well
